kolibri/utils/server.py

```diff
@@ -76,6 +81,19 @@ def rm_pid_file():
     run_server(port=port)
 
 
+class PingbackThread(threading.Thread):
+
+    @classmethod
+    def start_command(cls):
+        thread = cls()
+        thread.daemon = True
+        thread.start()
+
+    def run(self):
+        command = ManagementUtility().fetch_command("ping")
+        command.execute(server="http://telemetry.learningequality.org/api/v1/pingback", interval=0.2, checkrate=0.1)
```

Comment:
These are test parameters for now (with frequent pings, ever 12 seconds); we'll move these out from here and set to saner defaults before merging.